src/Font.cpp

```diff
@@ -507,7 +507,7 @@ void Font::LoadFontPageSettings( FontPageSettings &cfg, IniFile &ini, const RStr
 
 				wchar_t c;
 				if( sCodepoint.substr(0, 2) == "U+" && IsHexVal(sCodepoint.substr(2)) )
-					sscanf( sCodepoint.substr(2).c_str(), "%x", &c );
+					sscanf( sCodepoint.substr(2).c_str(), "%lc", &c );
```

Comment:
%lc is used for wide characters. BUT I just noticed that it's the address of c, and not c. Regardless, it should be %p for pointer instead of %x. %x assumes it's a an unsigned int, which won't be accurate for 64-bit systems.